### DISCUSSION TITLE
Есть ли четкий стиль для описания свойств / методов в доке?

### DISCUSSION THREAD

**Discussion by scffs** [VKCOM/VKUI#5197] [Q&A] [answered]
Поясню: какие-то свойства описаны полностью на английском, в каких-то нет точки в конце предложения, некоторых свойств нет в файле компонента вообще. 

Считать ли хоть что-то из этого багом или нет?

**Comment by inomdzhon** [comment, marked as answer]
За описание нативных для HTMLElement свойств и методов, что лежат под "Показать все свойства"

<img width="320" alt="image" src="https://github.com/VKCOM/VKUI/assets/5850354/57626040-8c69-49c1-b44b-024f46541102">

отвечает **React**, и ребята из **React** пишут свои описания на английском.

Свои кастомные свойства и методы мы описываем на русском языке, потому что у нас коммьюнити русскоязычное. Конечного пользователя обычно интересуют только кастомные свойства и методы. Нативные свойства и методы обычно и так знакомы.

Нет, это мы не считаем багом :)
